apps/web/src/services/cost-explorer/store/cost-analysis-page-store.ts

```diff
@@ -178,15 +158,15 @@ export const useCostAnalysisPageStore = defineStore('cost-analysis-page', {
                     name,
                     options,
                 });
-                this.selectedQueryId = createdData.cost_query_set_id;
+                costAnalysisLNBStore.$patch({ selectedQueryId: createdData.cost_query_set_id });
```

Comment:
praise: Oh, I didn't know that we can use pinia store inside the other pinia store. 😮
Thank you for introducing me to a new way of using it.